### ISSUE TITLE
Validate the IP address before doing query work

### ISSUE THREAD

**Issue by peter279k** [DivineOmega/php-geolocation#4]
As title, I think we should validate the IP address before passing `$ip` variable to do the query.

How about using this [package](https://github.com/darsyn/ip) to validate IP address? Thanks.

**Comment by Jord-JD** [comment]
Good idea, but perhaps it could be implemented with PHP's built-in `filter_var` method?

```php
if (filter_var($ip, FILTER_VALIDATE_IP)) {
    echo 'IP is valid.';
}

**Comment by peter279k** [comment]
@DivineOmega, thank you for your reply. I think it's okay to use the `filter_var` to validate the IP address.
BTW, does the location service provider support IPv4 and IPv6?

**Comment by Jord-JD** [comment]
Fixed as of v2.0.0.
